kernel.spec.in

```diff
@@ -499,11 +506,12 @@ mv  %buildroot/lib/firmware-all %buildroot/lib/firmware/%kernelrelease
 
 # Prepare initramfs for Qubes VM
 mkdir -p %buildroot/%vm_install_dir
+touch /tmp/dummy_conf
```

Comment:
Please either put that into owner-only writable dir, or generate using `mktemp` (which takes care about not reusing potentially malicious pre-existing file).

And also, remove it when no longer needed (after the dracut call).

Reply:
Maybe put it into `%_builddir` instead of `/tmp`? Then it will be automatically cleaned up too.